gprofiler/utils.py

```diff
@@ -97,12 +104,28 @@ def start_process(cmd: Union[str, List[str]], via_staticx: bool, **kwargs) -> Po
             env = env if env is not None else os.environ.copy()
             env.update({"LD_LIBRARY_PATH": ""})
 
+    cur_preexec_fn = kwargs.pop("preexec_fn", os.setpgrp)
+
+    if term_on_parent_death:
+        def wrap_with_set_pseg_death(inner):
```

Comment:
Need not be defined as a closure (only the inner one). Define it outside of `start_process`

Reply:
done